rts/Lua/LuaSyncedCtrl.cpp

```diff
@@ -1608,10 +1610,12 @@ int LuaSyncedCtrl::CreateUnit(lua_State* L)
 
 /***
  * @function Spring.DestroyUnit
+ * @see Spring.CreateUnit
  * @number unitID
  * @bool[opt=false] selfd makes the unit act like it self-destructed.
  * @bool[opt=false] reclaimed don't show any DeathSequences, don't leave a wreckage. This does not give back the resources to the team!
  * @number[opt] attackerID
+ * @bool[opt=false] cleanupImmediately stronger version of reclaimed, removes the unit unconditionally and makes its ID available for immediate reuse (otherwise it takes a few frames)
```

Comment:
```suggestion
 * @bool[opt=false] cleanup Immediately stronger version of reclaimed, removes the unit unconditionally and makes its ID available for immediate reuse (otherwise it takes a few frames)
```

Reply:
Won't that mean the argument becomes called "cleanup" and its description becomes "Immediately stronger ..."? The point is that the cleanup is immediate.

Reply:
Ah, I read it wrong lol